s3_to_redshift.go

```diff
@@ -44,9 +46,20 @@ func fatalIfErr(err error, msg string) {
 	}
 }
 
+// rounds down a dateTime to the timeGranularity
+func truncateDate(date time.Time) time.Time {
```

Comment:
Also when you say "rounds down..." does it literally round down? For example does 11:50 round to 11:00 or 12:00?